recipe/meta.yaml

```
@@ -72,15 +75,43 @@ outputs:
       source_files:
         - tests/
       requires:
+        - pip
         - mock
         - pytest
         - scipy
-        - pip
-        - jax >=0.4  # [osx or linux64]
+        - dill
+        # Recent versions of jax are not available on Python 3.8
+        # https://github.com/conda-forge/jax-feedstock/pull/123
+        # jax is not properly working in linux-aarch64
+        # https://github.com/conda-forge/jax-feedstock/issues/144
+        - jax >=0.4  # [unix and not ppc64le and not aarch64 and py != 38]
+        - flax  # [unix and not ppc64le and not aarch64 and py != 38]
```

I think this might be easier to digest
```suggestion
        - jax >=0.4      # [unix and not (aarch64 or ppc64le or py==38)]
        - flax           # [unix and not (aarch64 or ppc64le or py==38)]
```